bitmovin/analytics-google-datastudio-connector
# AN-2452 timzones support

How to test
- ping me for report 
- you will see two charts but the data is kinda shifted by one day
- click on one chart and edit the datasource
- set the timezone to `+02:00` (same as license) and save
- the two charts` data should align again
